lib/spack/spack/installer.py

```diff
@@ -280,6 +280,32 @@ def _print_installed_pkg(message):
     print(colorize("@*g{[+]} ") + spack.util.path.debug_padded_filter(message))
 
 
+def _print_install_test_log(pkg, verbose):
+    """Output install test log file information.
+
+    Args:
+        pkg (spack.package.PackageBase): package of interest
```

Comment:
Can you move these types from the docstring to the function signature so that mypy can test them?

Reply:
Ouch.  I keep forgetting we changed `package` module to `package_base` too.

Reply:
Grrr.  I keep getting `AttributeError: module 'spack' has no attribute 'package_base'`.

Reply:
I can add `: bool` but it is refusing to work when I add `: spack.package_base.PackageBase`.

Reply:
You might have to import package_base

Reply:
It is already importing `spack.package_base`.

Reply:
The module is already used for several calls (and docstrings).

Reply:
Try adding the commit so I can see the failing tests

Reply:
Will do.  I'm manually testing colorizing test log messages (and `Completed testing`).

Reply:
Done

Reply:
Interesting.  My local checks didn't add the comment that it was most likely due to a circular import.

Reply:
Can I revert that last commit and add the typing to the refactored code in #34236?

Reply:
If you put the base class in quotes (so it's only evaluated by the type checker and doesn't actually need to be imported) does that work?

Reply:
I didn't realize that was an option.  🎉